mixins/eirini/templates/metrics-service-account.yml

```diff
@@ -7,6 +7,10 @@ kind: ServiceAccount
 metadata:
   name: eirini-metrics
   namespace: {{ .Release.Namespace }}
+{{- if .Values.kube.image_pull_secrets }}
+imagePullSecrets:
+{{- list . "" | include "kubecf.imagePullSecrets" }}
+  {{- end }}
```

Comment:
I would rather see the middle line indented than the closing `{{- end }}` 😄 (Yeah, I know this was just a typo)

Reply:
confirmed